unit_test_candidates.md

```
@@ -0,0 +1,140 @@
+# Unit Test Candidates for DrevOps Behat Steps
+
+This document identifies methods and code sections within the DrevOps Behat Steps traits that are good candidates for unit testing.
+
+## Selection Criteria
+Methods identified meet one or more of the following criteria:
+- Pure functions without external dependencies
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)